deployments/add-ons/prometheus/README.md

```diff
@@ -0,0 +1,57 @@
+## Why you should use Prometheus with Amazon Managed Service for Prometheus (AMP)
+Many Kubeflow users utilize Prometheus and Grafana to monitor and visualize their metrics. However it can be difficult to scale open source Prometheus and Grafana as the number of nodes to be monitored increases. AMP seeks to simplify this issue by allowing multiple Prometheus servers to aggregate their metrics in Amazon Managed Prometheus, and finally Amazon Managed Grafana allows customers to then view these aggregated metrics.
+
+## Prerequisites
+First download one of our deployment options by following the directions at: https://awslabs.github.io/kubeflow-manifests/docs/deployment/prerequisites/
+
+**Note:** The steps below will assume you are sitting in the kubeflow-manifests directory.
+
+## Steps to Setup Prometheus and AMP
+1. Export cluster name and region as environment variables:
+    1. `export CLUSTER_NAME=<your-cluster-name>`
```

Comment:
Can we convert the inline code to code blocks:

E.g.
```
export CLUSTER_NAME=<your-cluster-name>
```

I think the inline blocks mess up the `--` dashes which makes the commands fail when you copy paste.